boltzrpc/boltzrpc.proto

```diff
@@ -407,6 +416,14 @@ message RefundSwapRequest {
     }
 }
 
+message ClaimSwapRequest {
```

Comment:
What about allowing multiple swaps to be claimed to the same address or wallet in one transaction?

Reply:
Can be done

Reply:
> What should the response be in that case? All swapinfos or just the claim transaction id

ID and claim transaction id should be enough.

> Its a inconvenient since it should probably align with refundswap which is already included in the last release

Yes, but let's not constrain ourselves by past mistakes. I'd rather have them be different than both not optimal